pages/passport/barriers/identify-a-barrier.tsx

```diff
@@ -121,22 +119,24 @@ const IdentifyABarrier: React.FunctionComponent = () => {
               label={intl.formatMessage({
                 defaultMessage: "Barrier description",
               })}
+              rules={{
+                required: intl.formatMessage(errorMessages.required),
+              }}
               rows={10}
             />
           </div>
-          <p data-h2-margin="b(top-bottom, l)" data-h2-font-weight="b(600)">
-            {intl.formatMessage({
-              defaultMessage:
-                "Or, use the worksheet to select a barrier (only your barrier information will be saved).",
-            })}
+          <p>
+            {" "}
+            {intl.formatMessage(
+              {
+                defaultMessage:
+                  "<strong>Use the tool to select a barrier </strong> (a barrier means anything that prevents you from participating in your work environment). You can select the barriers from the dropdown menus below.",
```

Comment:
"You can" must be wrapped in a strong tag as well according to the docs:
![image](https://user-images.githubusercontent.com/22059495/170766132-4519746c-270c-4388-944c-cdf4e441151b.png)